doc/development/development_guide.rst

```diff
@@ -127,24 +127,65 @@ Individual test modules are run by invoking pytest directly from the command lin
 
 .. note:: **Adding tests to Strawberry Fields**
 
-    The ``tests`` folder is organised into four subfolders: ``backend`` for tests that
-    only import a Strawberry Fields backend, ``frontend`` for tests that import the Strawberry
-    Fields UI but do not make use of a backend, ``integration`` for tests that test
-    integration of the frontend and backends, and ``apps`` for tests of the applications layer.
+    The ``tests`` folder is organised into several subfolders:
 
-    When writing new tests, make sure to mark what components it tests. For a backend test,
-    you can use the ``backends`` mark, which accepts the names of the backends:
+    - ``backend`` for tests that only import a Strawberry Fields backend,
+    - ``frontend`` for tests that import the Strawberry Fields UI but do not make use of a backend,
+    - ``integration`` for tests that test integration of the frontend and backends,
+    - ``apps`` for tests of the applications layer
```

Comment:
We will need to add `api` here for tests that only import and use the `strawberryfields.api` package!

Reply:
Thanks :)